frame/support/src/weights/block_weights.rs

```diff
@@ -33,47 +34,93 @@
 //   --weight-path=./frame/support/src/weights/
 //   --warmup=10
 //   --repeat=100
+//   --header=HEADER-APACHE2
 
 use sp_core::parameter_types;
 use sp_weights::{constants::WEIGHT_PER_NANOS, Weight};
 
 parameter_types! {
 	/// Time to execute an empty block.
-	/// Calculated by multiplying the *Average* with `1` and adding `0`.
+	/// Calculated by multiplying the *Average* with `1.0` and adding `0`.
 	///
-	/// Stats nanoseconds:
-	///   Min, Max: 5_303_128, 5_507_784
-	///   Average:  5_346_284
-	///   Median:   5_328_139
-	///   Std-Dev:  41749.5
+	/// Statistics in nanoseconds:
+	///   Min, Max: 337_266, 439_477
```

Comment:
Somehow the block weights reduced a lot. This is also on master. It was fixed in Polkadot by this https://github.com/paritytech/substrate/pull/12232 but not here?!